go_templates/query.vm

```diff
@@ -271,6 +271,27 @@ func (s *${queryType}) Do(ctx context.Context, headers ...*common.Header) (respo
   err = s.c.get(ctx, &response, $processedPath, $queryArg, headers)
   return
 }
+#elseif( $queryType == "SetBlockTimeStampOffset" )## TODO: "post" can be generated, there is nothing special here.
+func (s *${queryType}) Do(ctx context.Context, headers ...*common.Header) (response $returnType, err error) {
+  err = s.c.post(ctx, &response, $processedPath, nil, headers, nil)
+  return
+}
+#elseif( $queryType == "SimulateTransaction" )
```

Comment:
I mostly modified the sendRawTransaction endpoint, but I'm not sure if this is correct. I did a quick sanity check in the Go SDK but only checked that it compiles correctly.

Reply:
Let's copy one of the simple implementations. Especially since this endpoint is using JSON instead of binary.

Reply:
Got it, I used the ~one liner implementation in [7dc1210](https://github.com/algorand/generator/pull/60/commits/7dc12103da3f28c357f188df5b8d39adffb7f79c)